### ISSUE TITLE
bootstrap.sh is not explained in developer's docs

### ISSUE THREAD

**Issue by rhaas80** [LLNL/scr#326]
The dev docs on https://scr-dev.readthedocs.io/en/latest/ do not seem to mention how to use `bootstrap.sh` to set up a SCR development copy.

**Comment by adammoody** [comment]
Good point, @rhaas80 .  I replaced the description we had about bootstrap.sh in the user guide with new steps about building from a release tarball.  We should add something about bootstrap.sh to the developer guide.
